### ISSUE TITLE
A11y_Model Builder_Create New Workspace_Status messages: Narrator/NVDA is not announcing status message/loading state that is displayed while creating a new workspace.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1595]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11y_Model Builder; #A11yWCAG2.1; #A11yTCS; #WCAG4.1.3; #Status messages; #NVDA; #Narrator; #Win32; #DesktopApp; #A11ySev2; #ModelBuilder-Win32-Jun2021; #Visual Studio Client;

### Environment Details:
Application name: Model Builder

### OS Details:
Windows Version 21H1 OS Build (19043.1083)
Microsoft Visual Studio 

### Note:

1. This issue is also observed while creating new experiment/new compute name. Loading state is not announced by screen reader.
2. This issue is also observed with NVDA

### Repro Steps:

1. Turn on Narrator/NVDA. Enable ML.NET Model builder as a preview feature in tools>options>Environment>preview features.
2. Create a .NET C# project and tab till project under solution explorer and Add machine learning.
3. Tab till Image classification under scenario tab and press enter.
4. Select Azure under environment tab and tab till set up workspace and press enter.
5. In 'Create New Experiment' dialog, tab till 'New' link beside Machine learning workspace name field and press enter
6. In 'Create New Workspace' fill all the required fields and tab till create button and press enter.
7. While creating a new workspace, a status message will be displayed, verify whether screen reader is announcing it or not.

### Actual Result:
Narrator/NVDA is not announcing status message/loading state that is displayed while creating a new workspace.

### Expected Result:
Narrator/NVDA should announce status message/loading state that is displayed while creating a new workspace.

### User Impact:
Screen reader users will not understand the status if NVDA/Narrator is not announcing what is displayed on screen

**Comment by beccamc** [comment]
Validate on rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Screen Reader is announcing status message that is displayed while creating a workspace. Hence closing the issue

https://user-images.githubusercontent.com/83570638/131447724-70b8f93c-5c3b-4906-adf6-552c15f7f924.mp4

**Comment by VijayaBharathyA** [comment]
#Closed;
